recipes/beyla-golden-signals/gke/beyla-daemonset.yaml

```diff
@@ -122,5 +122,9 @@ data:
     attributes:
       kubernetes:
         enable: true
+        drop_external: true
```

Comment:
would be good to add a comment explaining what the options are for

Reply:
done